contracts/math/SafeMath.sol

```
@@ -20,8 +20,10 @@ library SafeMath {
   }
 
   function sub(uint256 a, uint256 b) internal constant returns (uint256) {
-    assert(b <= a);
-    return a - b;
+    if(b <= a) {
+      return a - b;
+    }
+    revert();
```

can you rewrite this following the Fail Fast principle?
```
if (b > a) {
  revert();
}
return a - b;